src/lib_wasm.rs

```diff
@@ -5,12 +5,15 @@
 extern crate base64;
 
 use crate::{
-    rewriter::{generate_prefix_stmts, print_js, rewrite_js, Config},
+    rewriter::{generate_iast_prefix_stmts, print_js, rewrite_js, Config},
```

Comment:
By the way, this has been renamed bc there will be a function called generate_et_prefix_stmts then

Reply:
Done.